### ISSUE TITLE
Deep Learning

### ISSUE THREAD

**Issue by Brad-Edwards** [Brad-Edwards/algos#13]
1. Backpropagation  
2. Stochastic Gradient Descent (SGD)  
3. Adam Optimizer  
4. RMSProp  
5. AdaGrad  
6. RProp (resilient propagation)  
7. Dropout (regularization)  
8. Batch Normalization  
9. Convolution (core of CNNs)  
10. BPTT (Backprop Through Time, RNNs/LSTMs)
